index.html

```diff
@@ -158,8 +158,15 @@
            observer.disconnect();
       }
     });
-    // subscribe to Resource-Timing and User-Timing events
-    observer.observe({entryTypes: ['resource', 'mark', 'measure']});
+    observer.observe({
+      // subscribe to Resource-Timing, User-Timing, and Server-Timing events
+      entryTypes:
+        ['resource', 'mark', 'measure', 'server'],
+
+      // retrieve the buffered Resource-Timing, User-Timing, and Server-Timing events
+      bufferedTypes:
+        ['resource', 'mark', 'measure', 'server']
```

Comment:
You are suggesting that the `buffered:true` will return all of the buffered entries of all of the specified `entryTypes`?

I know that [no one cares about onload anymore](https://www.stevesouders.com/blog/2013/05/13/moving-beyond-window-onload/), but if an analytics script only wants to report on timing entries for blocking resources, with this approach, they wouldn't have to immediately disconnect to no longer be notified about resources post-onload. 

With `bufferedTypes`, they would write:
```
new PerformanceObserver(function(list, observer) {
  // process entries here
}).observe({
  bufferedTypes: ['resource']
})
```

Without: 
```
new PerformanceObserver(function(list, observer) {
  // process entries here
  observer.disconnect()
}).observe({
  entryTypes: ['resource']
})
```

Also, if blocking entries are all you care about, you wouldn't be able to know if an entry was before or after onload without comparing the timestamps to nav-timing.